package.json

```diff
@@ -22,7 +22,8 @@
   "dependencies": {
     "buffer-crc32": "^0.2.13",
     "fs-extra": "^6.0.0",
-    "klaw": "^2.1.1"
+    "klaw": "^2.1.1",
+    "yarn": "^1.7.0"
```

Comment:
`yarn` shouldn't be installed as a dependency.

Reply:
See: https://github.com/ExodusMovement/zip-up/pull/4